apps/dapp-console-api/src/constants/chains.ts

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import abitype from 'abitype'
+import type { PublicClient } from 'viem'
+import { createPublicClient, http } from 'viem'
+import type { Chain } from 'viem/chains'
+import { optimism } from 'viem/chains'
+
+const SUPPORTED_CHAINS: Chain[] = [optimism]
+
+export const supportedChainsPublicClientsMap = SUPPORTED_CHAINS.reduce(
```

Comment:
not something to solve right away, we're probably going to want to put all the superchain info in `op-app` or a common pkg that keeps track of all this info so we don't drift apart between all of our services and frontends